lib/src/styles/string_cons.dart

```diff
@@ -0,0 +1,7 @@
+abstract class StringConstant {
```

Comment:
Dont you think a better name would be `MovieStrings` ?

Reply:
Agreed. Fixed!